src/applications/hca/containers/AuthBenefitsPackagePage.jsx

```diff
@@ -35,19 +34,19 @@ const AuthBenefitsPackagePage = props => {
   const [localData, setLocalData] = useState({});
   const dispatch = useDispatch();
 
+  const setFormData = dataToSet => dispatch(setData(dataToSet));
```

Comment:
Was this updated as a readability change? It looks like functionally it is the same as before.